javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,166 @@
+    //  are firstname and surname are not empty and there values are not  false or true? 
+    if (arguments.length >= 2 && arguments[1] !== true && arguments[1] !== false) {
+        if (useFormalName === true && firstname !== '' && surname !== '')
+            return 'Lord ' + firstname + ' ' + surname;
+        else if (useFormalName === false && firstname !== '' && surname !== '')
+            return firstname + ' ' + surname;
+        else
+            return 'Please Enter First name and Last Name';
+    }
+    else
+        return 'Please Enter First name and Last Name';
+
+}
+
+let fullname1 = getFullname('Benjamin', 'Hughes', true);
+let fullname2 = getFullname('viktor', 'hogu');
+console.log('Fullname1 : "' + fullname1 + '" ');
+console.log('Fullname2 : "' + fullname2 + '"');
+
+
+/**Event application */
+function getEventWeekday(noOfDays) {
+    let date = new Date();
+    let todays = date.getDay();
+    let weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+    return weekDays[(todays + noOfDays % 7) % 7]
+}
+// With todays weekday a tuesday
+console.log(getEventWeekday(9)); // Logs out "Thursday"
+// With todays weekday a Friday
+console.log(getEventWeekday(2)); // Logs out "Sunday"
+
+/**Weather wear */
+function recommendToWear(temperature) {
+
+    if (temperature > 55) {
+        return "Are you at Venus? You don't need to wear any clothes ";
+    }
+    else if (temperature <= 55 && temperature > 40) {
+        return "Its too hot stay inside and wear shorts";
+    }
+    else if (temperature <= 40 && temperature > 30) {
+        return "Wear shorts and drink lots of water";
+
+    }
+    else if (temperature <= 30 && temperature > 20) {
+        return "Wear shorts and T-shirt";
+    }
+    else if (temperature <= 20 && temperature > 10) {
+        return "Wear summer jacket  ";
+    }
+    else if (temperature <= 10 && temperature > 0) {
+        return "Wear jacket and full size trouser";
+    }
+    else if (temperature <= 0 && temperature > -10) {
+        return "Wear winter jacket and trouser";
+    }
+    else if (temperature <= -10 && temperature > -20) {
+        return "Wear double winter jacket";
+    }
+    else if (temperature <= -20 && temperature > - 30) {
+        return "Wear very very warm clothes and stay inside";
+    }
+    else if (temperature <= -30) {
+        return "Are you at pluto? I can't suggest you any anything";
+    }
+    else {
+        return "Please Enter valid temprature";
+    }
+}
+
+let clothesToWear = recommendToWear(21);
+console.log(clothesToWear); // Logs out: "shorts and a t-shirt"
+
+/**addStudentToClass function */
+const class07Students = [];
+function addStudentToClass(studentName) {
+    const numberOfStudents = getNumberOfStudents()
+    /**check class has stil place for one student and student name is not Queen */
+    if (numberOfStudents >= 6 && studentName.toLowerCase() !== 'queen')
```

Comment:
what happened if we add queen twice when students are more than 6?

Reply:
I thought queen can also get in once like others  but now I have allow her to get in any time .